rapids-cmake/cython/set_lib_dirs.cmake

```diff
@@ -0,0 +1,53 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+include_guard(GLOBAL)
+
+#[=======================================================================[.rst:
+rapids_cython_set_lib_dirs
+--------------------------
+
+.. versionadded:: v22.12.00
+
+Set the directories where any dependent libraries will be placed.
+
+.. code-block:: cmake
+
+  rapids_cython_set_lib_dirs(<dir1> <dir2> ...)
+
+The relative path to the provided directories is appended to the rpath of any
```

Comment:
This comment doesn't seem to match what occurs.

This function looks to convert a set of paths if either abs or relative to all absolute.